scripts/run_e2e_tests.py

```diff
@@ -164,6 +163,48 @@
 # the oppia web server.
 SUBPROCESSES = []
 
+RERUN_POLICIES = {
+    'accessibility': 'never',
```

Comment:
Should we fetch this data from the flak-reporting server? that way we won't have to create a PR here to update these values all the time. Also, it makes sense that flake-server should report which suites are flaky. [But the other concern that we will increase the number of calls made on the Heroku server, wdyt?]

Reply:
I'm uncomfortable with having our rerun policies automatically change. There might be one-off events, like the recent case where all the tests were passing briefly, that don't represent the long-term trends these policies are trying to reflect. I'd prefer that the Automated QA team keep a human in the loop whenever these policies change